hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/TopologySubcommand.java

```diff
@@ -68,6 +69,13 @@
       description = "Print Topology with full node infos")
   private boolean fullInfo;
 
+  @CommandLine.Option(names = {"-n", "--nodeOperationalState"},
+      description = "Show info by datanode NodeOperationalState" +
```

Comment:
I think "show ... by" is ambiguous here, could mean grouping or filtering.  I suggest:

```suggestion
      description = "Only show datanodes in a specific operational state " +
```